web-app/js/portal/filter/ui/DateFilterPanel.js

```diff
@@ -120,26 +120,16 @@ Portal.filter.ui.DateFilterPanel = Ext.extend(Portal.filter.ui.BaseFilterPanel,
 
     _applyDateFilterPanel: function(component) {
```

Comment:
You probably didn't name this but this method name threw me off.  Would _applyDateFilter make more sense?